### ISSUE TITLE
validate-flutter-docs breakage

### ISSUE THREAD

**Issue by jdkoren** [dart-lang/dartdoc#2105]
The `validate-flutter-docs` build task seems to be broken ~~after #2096~~

EDIT: now I'm not sure since when this is failing. [Link](https://travis-ci.org/dart-lang/dartdoc/builds/628562356) to Travis.

**Comment by jdkoren** [comment]
It seems to fail on [webdriver.core](https://github.com/google/webdriver.dart/blob/master/lib/core.dart) and [webdriver.async_core](https://github.com/google/webdriver.dart/blob/master/lib/async_core.dart), though I'm not sure why...

**Comment by jdkoren** [comment]
Appears to be a duplicate of #2110
